debian/patches/netplan99-cannot-use-default.patch

```diff
@@ -0,0 +1,167 @@
+Description: Retain compatible route definition for netplan 0.99
+ focal contains netplan v0.99 which is not compatible
+ with Routes' definitions containing `to: default`. This patch
+ retains the previous way to define routes.
+Author: james.falcon@canonical.com
+Origin: backport
+Forwarded: not-needed
```

Comment:
Given this is tracked in a Github issue now, and we didn't represent that in d/changelog, we minimally want a link to the upstream issue here in dep3 header format
```suggestion
Bug: https://github.com/canonical/cloud-init/issues/4133 
Forwarded: not-needed
```

Reply:
Good idea. I just made sure we had tracking via these GitHub links, but this is better tracking.